standalone-server/package.json

```diff
@@ -27,31 +27,33 @@
   "style": "app.css",
   "dependencies": {
     "body-parser": "^1.15.0",
-    "browserify": "^16.0.0",
     "compression": "^1.7.1",
     "crypto": "0.0.3",
+    "css-loader": "^3.0.0",
+    "d3-fetch": "^1.1.2",
+    "d3-request": "^1.0.6",
     "express": "^4.13.4",
     "fs": "0.0.2",
     "multer": "^1.1.0",
-    "parcelify": "^2.2.1",
     "query-graphs": "file:../query-graphs",
     "recursive-readdir": "^2.2.1",
     "spin": "0.0.1",
-    "uglify-js": "^2.7.0"
+    "style-loader": "^0.23.1",
+    "uglify-js": "^2.7.0",
```

Comment:
Not familiar with webpack: do we still need the explicit dependency on uglify-js or is webpack production mode enough?